### PR TITLE
Default to no -S/-T arguments...

### PR DESCRIPTION
**Pull request by gquintard**
...but still leave the option to use them is specified in the reload
script.

### CODE REVIEW

File: redhat/varnish.sysconfig [hunk 1]
```diff
@@ -18,9 +18,7 @@ NPROCS="unlimited"
 # DAEMON_COREFILE_LIMIT="unlimited"
 
 # Init script support to reload/switch vcl without restart.
-# To make this work, you need to set the following variables
-# explicit: VARNISH_VCL_CONF, VARNISH_ADMIN_LISTEN_ADDRESS,
-# VARNISH_ADMIN_LISTEN_PORT, VARNISH_SECRET_FILE.
+# To make this work, you need to explicitly set VARNISH_VCL_CONF
```
Comment (Dridi) [inline redhat/varnish.sysconfig]: Missing a dot at the end of the sentence ;)

File: debian/reload-vcl [hunk 2]
```diff
@@ -117,10 +117,9 @@ then
     exit 1
 fi
 
-if [ -z "$mgmt_interface" ]
+if [ -n "$mgmt_interface" ]
 then
-    printf >&2 "$msg_no_management"
```
Comment (Dridi) [inline debian/reload-vcl]: You should remove msg_no_management too.

File: redhat/varnish_reload_vcl [hunk 3]
```diff
@@ -75,8 +67,20 @@ else
...
+
+else
+	echo "Error: only one of VARNISH_ADMIN_LISTEN_ADDRESS or VARNISH_ADMIN_LISTEN_PORT was defined"
+        echo "       Both or none must be specified"
+	exit 2
+fi
+
 # Done parsing, set up command
-VARNISHADM="varnishadm $secret -T $VARNISH_ADMIN_LISTEN_ADDRESS:$VARNISH_ADMIN_LISTEN_PORT"
+VARNISHADM="varnishadm $secret $mgmt_interface"
```
Comment (Dridi) [inline redhat/varnish_reload_vcl]: I would simply get rid of the VARNISHADM variable and remnants of the cli/secret setup, this is meant for local service reloads so having root privileges should be enough.

**Comment by denisbr** [comment]
Does this work with both VC 4.1 / 5.x? @gquintard ?

**Comment by Dridi** [comment]
@denisbr regarding `-T`, yes. However I don't remember since when the lack of `-S` means a random secret. It could have been from the beginning.

**Comment by Dridi** [comment]
It works with 4.1.2, I assume it works with all 4.1 releases.

**Comment by Dridi** [comment]
Let's revisit this after #72 is done, so we don't bother changing the reload scripts.

**Comment by Dridi** [comment]
From @denisbr in #81:

> This sounds good for the 6.0 release and future releases, but we should keep the old way for the older packages imo.

I'm a bit wary of ending up with more conditionals.

**Comment by Dridi** [comment]
Superseded  by #94, thank you both @gquintard and @denisbr.

**Pull request closed without merging**